ci/audit.yml

```diff
@@ -0,0 +1,15 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: onsdigital/dp-concourse-tools-nancy
+    tag: latest
+
+inputs:
+  - name: dp-reporter-client
```

Comment:
I think the copy paste hasn't helped you this morning.

Double check the repo names, should this be `dp-elasticsearch`

Reply:
Check the other yml's too

Reply:
Git woes.  Pushed again.